app/src/main/java/com/example/deejai/RoomSelection/RoomAdapter.kt

```diff
@@ -37,6 +41,15 @@ class RoomAdapter(val rooms : List<Room>) : Adapter<RoomAdapter.RoomViewHolder>(
             roomName = itemView.findViewById(R.id.room_name)
             roomDistance = itemView.findViewById(R.id.room_distance)
             roomImage = itemView.findViewById(R.id.room_image) as ImageView
+            itemView.setOnClickListener(this)
+        }
+
+        override fun onClick(v: View?) {
+            val position = adapterPosition
+            val myintent = Intent(c, RoomPlayer::class.java)
+            Log.d("adapter", roomName.text.toString())
```

Comment:
Is this a debugging log? Please remove it.

Reply:
done